mempool/reactor.go

```diff
@@ -113,7 +124,14 @@ func (memR *MempoolReactor) broadcastTxRoutine(peer Peer) {
 		return
 	}
 
+	batchSize := memR.config.TxBatchSize
+	if batchSize <= 0 {
```

Comment:
this is usually being handled during config parsing.

Reply:
where/how? is there a nice way to enforce constraints on config values when loaded?

Reply:
I was thinking about some function like `Config#Parse()`

Reply:
added this to https://github.com/tendermint/tendermint/issues/628